### PR TITLE
Reflect recent src relocations

### PR DESCRIPTION
**Pull request by kevans91**
r325826 relocated sys/boot/fdt/dts to sys/dts
r325834 relocated sys/boot to stand

### CODE REVIEW

File: board/Alix/setup.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ alix_partition_image ( ) {
         # TODO: This is broken; should use 'make install' to copy
         # bootfiles to workdir, then install to disk image from there.
```
Comment (bsdimp) [inline board/Alix/setup.sh]: It would be better to fix this todo and use installed files not files plucked from objdir.

**Comment by bsdimp** [comment]
These changes are fine, but I'm surprised by the number of locations that needed to change in the source.

**Comment by adriaandegroot** [comment]
This change:

```
-    _FDTDIR=$FREEBSD_SRC/sys/boot/fdt/dts
 +    _FDTDIR=$FREEBSD_SRC/dts
```

strips `sys/` out as well, and it shouldn't. (Sorry, can't comment on the commit / line number exactly).

**Comment by brd** [comment]
Cherry-picked, thanks!

**Pull request closed without merging**